README.md

```diff
@@ -32,3 +32,23 @@ Use of the `--dump-hashes` option is recommended as it avoids having to
 read the file system again if you need to re-run the command.  See
 `--read-hashes` option.  Also the dump file must be provided when users
 request help.
+
+#### Note
+
+The first `dd` command is meant to be used when trying to restore the
+filesystem on a different support or partition rather than the damaged one.
+This will copy the first portion of the damaged partition, i.e. the portion
+that was already succesfully moved, stopping right before the overlapping
+section would begin.
+
+If the goal is to restore the filesystem while operating on the damaged
+partition itself, i.e. continuing the interrupted move/resize operation
+from gparted or a similar tool, the second `dd` command is sufficient.
+This will copy the data starting from where the overlap section ends (i.e.
+the data still to be moved) and move them back at the beginning of the
+overlap section itself. In this case, both `INPUT` and `OUTPUT` will point
+to the same disk partition you're tring to restore.
+
+**Warning**: always perform a backup of the partition you're trying to
+recover, even if damaged, before issuing any command. `dd` is a powerful
+tool and if used incorrectly will result in permanent data loss.
```

Comment:
This second and third paragraphs while not incorrect, they are risky and may lead to worse data loss.  The original case of writing `find-overlap` was a GParted resize/move failing because of a faulty sector on a hard drive [[SOLVED] Unallocated space and corrupted NTFS after resize](http://gparted-forum.surf4.info/viewtopic.php?id=18089).  In such a case even taking a backup is difficult and subject to failure.  If a user is lucky just using the second `dd` command on a fault hard drive will fail and not make it worse.

Reply:
Answered [below](https://github.com/mfleetwo/find-overlap/pull/1#issuecomment-2522684361).